clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -657,7 +657,7 @@ public void close(Duration timeout) {
                 // Wait for the thread to be joined.
                 thread.join(waitTimeMs);
             }
-            log.debug("Kafka admin client closed.");
+            log.info("Kafka admin client closed.");
```

Comment:
I wonder if changing this log level is really necessary. What's the rational behind it?

Reply:
I think it is helpful to get at least one log message at info level in shutdown. Really at the start of shutdown is best since it triggers logic which can be difficult to explain if you don't know shutdown is taking place. Anyway, let me remove this change here and I can open a separate patch so that we can do this consistently in all the clients.